src/EditorFeatures/CSharpTest/Completion/CompletionProviders/EnumAndCompletionListTagCompletionProviderTests.cs

```diff
@@ -29,15 +29,26 @@ static void Main(string[] args)
         Colors c = Colors.Blue;
     }
 }
- 
+";
+            var colors = @"
 enum Colors
 {
     Red,
     Blue,
     Green,
 }
 ";
-            await VerifyItemExistsAsync(markup, "Colors");
+            var colorsLike = @"
+readonly struct Colors
```

Comment:
Given the implementation covers both fields and properties, might consider adding a type defining static get-only properties.

Reply:
This should be covered by `TestProperties()`:

https://github.com/dotnet/roslyn/blob/3475981a50c03c5bb060c4abcaa3518fdf6fbcfb/src/EditorFeatures/CSharpTest/Completion/CompletionProviders/EnumAndCompletionListTagCompletionProviderTests.cs#L1916-L1938